server/telemetry.go

```diff
@@ -4,7 +4,6 @@ const (
 	telemetryOauthModeOauth             = "Oauth"
 	telemetryOauthModeOauthAccountLevel = "Oauth Account Level"
 
-	telemetryStartSourceWebapp  = "webapp"
```

Comment:
Why is this removed?

Reply:
This was not used anywhere and was giving linting error